app/boards/arm/puchi_ble/Kconfig.defconfig

```diff
@@ -0,0 +1,37 @@
+# Electronut Labs Papyr board configuration
+
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if BOARD_PUCHI_BLE_v1
+
+config BOARD
+	default "puchi_ble"
+
+if USB_DEVICE_STACK
+
+config USB_NRFX
+	default y
+
+endif # USB_DEVICE_STACK
+
+config BT_CTLR
+	default BT
+
+config ZMK_BLE
+	default y
+
+config ZMK_USB
+	default y
+
+config PINMUX
+	default y
+
+if BOARD_PUCHI_BLE_v1
+
+config BOARD_PUCHI_BLE_CHARGER
```

Comment:
I don't believe this section is necessary, board specific Kconfig will automatically get included.